Apps/Contoso.WPF.Puppet.DotNetCore/App.xaml.cs

```diff
@@ -45,6 +45,7 @@ protected override void OnStartup(StartupEventArgs e)
             }
 
             // Start AppCenter.
+            AppCenter.IsNetworkRequestsAllowed = AppCenter.IsNetworkRequestsAllowed;
```

Comment:
I might be missing something here due to lack of context highlight. Can you explain this line?